go.mod

```diff
@@ -3,22 +3,22 @@ module github.com/red-hat-storage/lvm-operator
 go 1.17
 
 require (
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v1.2.0
```

Comment:
- seeing CI pass is a good sign however pls do test locally once
- iirc some combo of logr and controller-runtime failed during the initial stages of the project

Reply:
@iamniting , I will merge this once you confirm that the local tests pass.